RadioMe/dist/metadata.json

```diff
@@ -15,9 +15,9 @@
       "radiome.fr"
   ],
   "version": "1.0",
-  "logo": "https://cdn.discordapp.com/attachments/534018310128402435/660519124732084244/logo.png",
-  "thumbnail": "https://cdn.discordapp.com/attachments/534018310128402435/660518138735099904/thumbnail.png",
+  "logo": "https://imgur.com/a/CGp1Gno",
+  "thumbnail": "https://imgur.com/a/HcPiBTX",

```

Comment:
bro you need to directly link the image

Reply:
you need the .png/.jpg links (Should also be i.imgur.com then I think)

Reply:
I'm sorry, I forgot that part.